### ISSUE TITLE
Minecraft container fails to pass MAX_MEMORY environment variable

### ISSUE THREAD

**Issue by InfiniteBed** [truenas/apps#630]
The Minecraft container does not recognize the `MAX_MEMORY` key, even though it is provided. This key worked in previous TrueNAS versions, but not Electric Eel.
<img width="791" alt="Screenshot 2024-10-07 at 4 46 19 PM" src="https://github.com/user-attachments/assets/f0e1f756-f0a2-43a9-9ecd-c6513ed20a5d">
<img width="432" alt="Screenshot 2024-10-07 at 4 50 35 PM" src="https://github.com/user-attachments/assets/31e82235-4f23-459d-b6bf-c8b47baa9d10">

**Comment by stavros-k** [comment]
Shell in the container and type `env` if the `MAX_MEMORY` is there defined, there its up to the application to pick it up.

Please let me know, thanks!

**Comment by InfiniteBed** [comment]
I found that it was indeed passed in, for the application to pick it up. I tested on another minecraft instance and found it to be applying correctly. I don't quite understand why it decided to work now, but my guess is, if the container's allowed memory is larger than TrueNAS's imposed memory limit, it will reset to default. 

I may raise an issue towards itzg's container to push an error when something like that happens. Regardless, I'll investigate further. Thank you for your efforts.
